install/kubernetes/helm/meshery/charts/meshery-cpx/values.yaml

```diff
@@ -11,10 +11,10 @@ image:
 env: {}
 
 probe:
-  #TODO: Need to confirm the health check path of meshery. 
+  #TODO: Need to confirm the health check path of meshery.
```

Comment:
@navendu-pottekkat  what's Meshery's health check path?

Reply:
There is no specific health-check path but we can try hitting the version endpoint to get the surety that meshery is running properly.

Reply:
A good item to create a new issue for follow up here.